parl/remote/tests/get_set_attribute_test.py

```diff
@@ -0,0 +1,112 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+import parl
+import numpy as np
+from parl.remote.client import disconnect
+from parl.utils import logger
+from parl.remote.master import Master
+from parl.remote.worker import Worker
+import time
+import threading
+import random
+
+
+@parl.remote_class
+class Actor(object):
```

Comment:
add unittest cases:
1. `Actor` has the same attribute name and function name. try to get the attribute and call the function.
2. try to get a non-existing attribute whose name is the same as the function name.
3. try to call a non-existing function whose name is the same as the attribute names.

Reply:
added